virttest/utils_test/__init__.py

```diff
@@ -1859,7 +1860,8 @@ class Stress(object):
 
     def __init__(self, stress_type, params, download_url="", make_cmds="",
                  stress_cmds="", stress_args="", work_path="",
-                 uninstall_cmds="", download_type="tarball"):
+                 uninstall_cmds="", download_type="file",
+                 downloaded_file_path=None):
```

Comment:
@balamuruhans , the last question:
   we define downloaded_file_path=None in parent class "Stress", and we don't set it in both class VMStress and HostStress, like
  class VMStress(Stress):
    """
    Run Stress tool on VMs, such as stress, unixbench, iozone and etc.
    """

    def __init__(self, vm, stress_type, params, download_url="", make_cmds="",
                 stress_cmds="", stress_args="", work_path="",
                 uninstall_cmds="", download_type="file"):
So is that a design or a missing?

Reply:
@aliang123 indeed, its a miss. Thank you for figuring it out, I will fix it.